### PR TITLE
Player panel fix

### PR DESCRIPTION
**Pull request by farie82**
## What Does This PR Do
Fixed the issue where the wrong param was changed in the player panel. Now all buttons work as expected again.

Fixes issues left over from #14404

## Why It's Good For The Game
Regression bug fixed

## Changelog
:cl:
fix: The player panel is usuable again
/:cl:

### CODE REVIEW

File: code/modules/admin/player_panel.dm [hunk 1]
```diff
@@ -297,15 +297,15 @@
...
+			var/clientUID = M.client ? M.client.ckey : null
 			//output for each mob
 			dat += {"
 
 				<tr id='data[i]' name='[i]' onClick="addToLocked('item[i]','data[i]','notice_span[i]')">
 					<td align='center' bgcolor='[color]'>
 						<span id='notice_span[i]'></span>
 						<a id='link[i]'
-						onmouseover='expand("item[i]","[M_job]","[M_name]","[M_rname]","--unused--","[M_key]","[M.lastKnownIP]",[is_antagonist],"[M.ckey]","[clientUID]","[M_eyeUID]")'
+						onmouseover='expand("item[i]","[M_job]","[M_name]","[M_rname]","--unused--","[M_key]","[M.lastKnownIP]",[is_antagonist],"[M.UID()]","[clientUID]","[M_eyeUID]")'
```
Comment (farie82) [inline code/modules/admin/player_panel.dm:308]: Reset it to what it was:
https://github.com/ParadiseSS13/Paradise/pull/14404/files#diff-d3ef5d44eb8f582b2b1637d9c9d46d12R308

File: code/modules/admin/player_panel.dm [hunk 2]
```diff
@@ -297,15 +297,15 @@
 				var/mob/living/silicon/ai/A = M
 				if(A.client && A.eyeobj) // No point following clientless AI eyes
 					M_eyeUID = "[A.eyeobj.UID()]"
-			var/clientUID = M.client ? M.client.UID() : null
+			var/clientUID = M.client ? M.client.ckey : null
```
Comment (Kyep) [inline code/modules/admin/player_panel.dm:300]: Good old clientUID = client.ckey.

Comment (farie82) [reply code/modules/admin/player_panel.dm]: It works! But yeah I changed the var names to match the value better

**Review by AffectedArc07 — APPROVED** [review]
I swear if this breaks again

**Pull request merged**